userauth.go

```diff
@@ -202,55 +217,63 @@ func (u *ValidateFromToken) getjwtkey(jwtpubkeypath string) error {
 				return err
 			}
 			if info.Mode().IsRegular() {
-				log.Debug("reading file: ", filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
+				log.Debug("Reading file: ", filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
 				keyData, err := ioutil.ReadFile(filepath.Join(filepath.Clean(jwtpubkeypath), info.Name()))
 				if err != nil {
-					return fmt.Errorf("token file error: %s", err)
+					return fmt.Errorf("token file error: %v", err)
 				}
 				mapkey := re.FindStringSubmatch(info.Name())[1]
```

Comment:
```suggestion
				mapkey := re.FindStringSubmatch(info.Name())[1]
```

This one will generate `panic: runtime error: index out of range [1] with length 0` if there is no valid public key in the path

Reply:
Fixed.